communitylighting.js

```diff
@@ -38,7 +38,26 @@ class CommunityLighting {
         });
 
         Handlebars.registerHelper(CommunityLighting.handlebarsHelpers);
-        
+
+        const baseSoundLoad = Sound.prototype.load;
+
+        Sound.prototype.load = async function load(...args) {
+            console.log(this);
+            // Call base fn
+            await baseSoundLoad.call(this, ...args);
+            if (CLAudioReactor.gainShim) {
+                this.container.gainNode.disconnect()
+                this.container.gainNode.connect(CLAudioReactor.gainShim)
+            }
+        };
+
+        PointSource.prototype.animate = function animate(dt) {
```

Comment:
Slightly worried about monkeypatching without calling the original function, is there any way we can run it like the soundLoad version?

I will investigate this